aiqleads/tests/core/test_project_tracking.py

```
@@ -3,16 +3,21 @@
 import pytest
+import shutil
 from datetime import datetime
 from typing import Dict
+from pathlib import Path
 
 from aiqleads.core.project_tracking import ProjectTracker, get_tracker
 
 class TestProjectTracking:
     @pytest.fixture
-    def tracker(self) -> ProjectTracker:
+    def tracker(self, tmp_path) -> ProjectTracker:
```

Ruff (0.8.2)

Missing type annotation for function argument `tmp_path`

(ANN001)